README.md

```diff
@@ -181,207 +182,327 @@ Switch to show netgroup section in nscd.conf. Allows for boolean, 'true', or 'fa
+Settings for enable_db_auth_attr in nscd.conf. Allows for boolean, 'true', or 'false'.
+
+- *Default*: 'USE_DEFAULTS'
+
+enable_db_bootparams
+---------------------
+Settings for enable_db_bootparams in nscd.conf. Allows for boolean, 'true', or 'false'.
+
+- *Default*: 'USE_DEFAULTS'
+
+enable_db_ethers
+-----------------
+Settings for enable_db_ethers in nscd.conf. Allows for boolean, 'true', or 'false'.
+
+- *Default*: 'USE_DEFAULTS'
+
+enable_db_exec_attr
+--------------------
+Settings for enable_db_exec_attr in nscd.conf. Allows for boolean, 'true', or 'false'.
+
+- *Default*: 'USE_DEFAULTS'
+
+enable_db_ipnodes
+------------------
+Settings for enable_db_ipnodes in nscd.conf. Allows for boolean, 'true', or 'false'.
+
+- *Default*: 'USE_DEFAULTS'
+
+enable_db_netmasks
+-------------------
+Settings for enable_db_netmasks in nscd.conf. Allows for boolean, 'true', or 'false'.
+
+- *Default*: 'USE_DEFAULTS'
+
+enable_db_networks
+-------------------
+Settings for enable_db_networks in nscd.conf. Allows for boolean, 'true', or 'false'.
+
+- *Default*: 'USE_DEFAULTS'
+
+enable_db_printers
+-------------------
+Settings for enable_db_printers in nscd.conf. Allows for boolean, 'true', or 'false'.
+
+- *Default*: 'USE_DEFAULTS'
+
+enable_db_prof_attr
+--------------------
+Settings for enable_db_prof_attr in nscd.conf. Allows for boolean, 'true', or 'false'.
+
+- *Default*: 'USE_DEFAULTS'
+
+enable_db_project
+------------------
+Settings for enable_db_project in nscd.conf. Allows for boolean, 'true', or 'false'.
+
+- *Default*: 'USE_DEFAULTS'
+
+enable_db_protocols
+--------------------
+Settings for enable_db_protocols in nscd.conf. Allows for boolean, 'true', or 'false'.
+
+- *Default*: 'USE_DEFAULTS'
+
+enable_db_rpc
+--------------
+Settings for enable_db_rpc in nscd.conf. Allows for boolean, 'true', or 'false'.
+
+- *Default*: 'USE_DEFAULTS'
+
+enable_db_tnrhdb
+-----------------
+Settings for enable_db_tnrhdb in nscd.conf. Allows for boolean, 'true', or 'false'.
+
+- *Default*: 'USE_DEFAULTS'
+
+enable_db_tnrhtp
+-----------------
+Settings for enable_db_tnrhtp in nscd.conf. Allows for boolean, 'true', or 'false'.
+
+- *Default*: 'USE_DEFAULTS'
+
+enable_db_user_attr
+--------------------
+Settings for enable_db_user_attr in nscd.conf. Allows for boolean, 'true', or 'false'.
+
+- *Default*: 'USE_DEFAULTS'
+
 enable_opt_auto_propagate
-----------------------
+-------------------------
 Settings for enable_opt_auto_propagate in nscd.conf. Allows for boolean, 'true', or 'false'.
 
 - *Default*: 'USE_DEFAULTS'
 
 passwd_enable_cache
-----------------------
-Settings for enable-cache passwd in nscd.conf. Must be 'yes' or 'no'.
+-------------------
+Settings for enable-cache service in nscd.conf where service can be either passwd, group, hosts, services, netgroup, audit_user, auth_attr, bootparams, ethers, exec_attr, ipnodes, netmasks, networks, printers, prof_attr, project, protocols, rpc, tnrhdb, tnrhtp, user_attr. Must be 'yes' or 'no'.
```

Comment:
Please don't bring back the old bad readable documentation with this PR.
Same is true for the following parts.